crates/accelerate/src/pauli_lindblad_map/qubit_sparse_pauli.rs

```diff
@@ -616,6 +616,43 @@ impl QubitSparsePauli {
             boundaries: vec![0, self.paulis.len()],
         }
     }
+
+    // Check if self commutes with other
```

Comment:
```suggestion
    // Check if `self` commutes with `other`
```
consider this change all over the place, depending on whether this is what they do in qiskit